workers/ohsome_quality_analyst/oqt.py

```diff
@@ -313,15 +330,18 @@ async def _(parameters: ReportBpolys, *_args) -> Report:
     report = report_class(feature=feature)
     report.set_indicator_layer()
 
+    tasks: List[Coroutine] = []
     for indicator_name, layer_name in report.indicator_layer:
-        indicator = await create_indicator(
-            IndicatorBpolys(
-                name=indicator_name,
-                layerName=layer_name,
-                bpolys=feature,
+        tasks.append(
+            create_indicator(
+                IndicatorBpolys(
+                    name=indicator_name,
+                    layerName=layer_name,
+                    bpolys=feature,
+                )
             )
         )
-        report.indicators.append(indicator)
+    report.indicators = await asyncio.gather(*(sem_task(task) for task in tasks))
```

Comment:
See comments before. Can't you integrate `sem_task(…)` into the `tasks` list directly?

Reply:
Yeah thats better. I did as you suggested to all usages of `sem_task()`. Thanks for pointing out.